staging/src/k8s.io/client-go/rest/config.go

```diff
@@ -420,5 +420,43 @@ func AnonymousClientConfig(config *Config) *Config {
 		QPS:           config.QPS,
 		Burst:         config.Burst,
 		Timeout:       config.Timeout,
+		Dial:          config.Dial,
+	}
+}
+
+// CopyConfig returns a copy of the given config
+func CopyConfig(config *Config) *Config {
```

Comment:
qualify name since some fields are explicitly not copied

Reply:
`CopyThreadableConfigFields`?  I could copy the rest I suppose. I'm just not completely certain of side-effects.  Are you?

Reply:
started copying everything